src/parsers/directive.parser.ts

```diff
@@ -18,7 +18,10 @@ export class DirectiveParser extends AbstractTemplateParser implements ParserInt
 		template = this._normalizeTemplateAttributes(template);
 		$(template)
 			.find('[translate]')
-			.each((i: number, element: CheerioElement) => {
+            .contents() // get child nodes
+            .filter(function() { // only keep text nodes
+				return this.nodeType === 3; // node type 3 = text node
```

Comment:
Hmm I wonder if this is the best approach. It means you cannot have *any* HTML in your keys. I can think of times Where you'd want to use <div>Hello <b>Username</b></div> or similar. Shouldn't it ultimately be the devs responsibility not to add the attribute to tags with lots of content?

In your example you should just use the pipe or wrap the text you actually want translated:

```
<div>
  <span class="icon">&times;</span>
  <span translate>Completed</span>
</div>
```